lib/jablotron.js

```diff
@@ -74,8 +73,11 @@ Jablotron.prototype = {
             if (sessionId && sessionId != null) {
                 self.client.doAuthenticatedRequest('/dataUpdate.json', payload, sessionId, function (response) {
                     let responseData = self.parseResponseData(response);
-                    if (responseData != null) {
-                        let segments = responseData[0]['data']['segments'];
+                    if (responseData["data"] != null) {
+                        if (responseData["service_events_type"] == "tamper" || responseData["service_events_type"] == "alarm") {
```

Comment:
Be carefull with these attributes - they could be null for thermometers. Even though thermometers use the same structure majority of attributes returned is null

Reply:
I assume you mean `if (responseData["service_events_type"]` ? This is on section level, I don't think it should influence the operation. 

Do you still have the thermometer for testing?

Reply:
Ok, I will update the other PR for merging, can you tomorrow check if it works?

Reply:
Yep, no problem